num2words/lang_RU.py

```diff
@@ -136,52 +148,49 @@ def setup(self):
                      "восемь": "восьмой",
                      "девять": "девятый",
                      "сто": "сотый"}
-        self.ords_feminine = {"один": "",
-                              "одна": "",
-                              "две": "двух",
-                              "три": "трёх",
-                              "четыре": "четырёх",
-                              "пять": "пяти",
-                              "шесть": "шести",
-                              "семь": "семи",
-                              "восемь": "восьми",
-                              "девять": "девяти"}
-
-    def to_cardinal(self, number):
+        self.ords_adjective = {"один": "",
+                               "одна": "",
+                               "две": "двух",
+                               "три": "трёх",
+                               "четыре": "четырёх",
+                               "пять": "пяти",
+                               "шесть": "шести",
+                               "семь": "семи",
+                               "восемь": "восьми",
+                               "девять": "девяти"}
+
+    def to_cardinal(self, number, gender='m'):
         n = str(number).replace(',', '.')
         if '.' in n:
             left, right = n.split('.')
             leading_zero_count = len(right) - len(right.lstrip('0'))
```

Comment:
There is an issue https://github.com/savoirfairelinux/num2words/issues/482

so I suggest, we need to avoid "запятая ноль ноль" so you could use 

```
            if set(right) == {'0'}:
                leading_zero_count = 0
            else:
                leading_zero_count = len(right) - len(right.lstrip('0'))
                
```
or something more beautiful